### PR TITLE
chore(ci): add automated releases

### LINKED ISSUE #15: Have official release binaries
Is it possible to cut a release with compiled binaries?

### PR DESCRIPTION
**Pull request by kurtisvg**
Fixes #15

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      steps:
+        - uses: actions/checkout@v3
+        - uses: wangyoucao577/go-release-action@v1.34
+          with:
+            release_tag: ${{ needs.release-please.outputs.release_tag }}
+            goos: ${{ matrix.goos }}
+            goarch: ${{ matrix.goarch }}
+            github_token: ${{ secrets.GITHUB_TOKEN }}
+
+
```
Comment (shubha-rajan) [inline .github/workflows/release.yaml:54]: nit: missing newline

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          releaseType: simple
+          versionFile: 'cmd/version.txt'
+  releases:
+      needs: release-please
+      if: ${{ needs.release-please.outputs.releases_created }}
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          # build and publish in parallel: linux/386, linux/amd64, linux/arm64, darwin/amd64, darwin/arm64
+          goos: [linux, darwin]
```
Comment (shubha-rajan) [inline .github/workflows/release.yaml:40]: would we want to publish windows images too? or does btlr only support linux and mac?

**Review by shubha-rajan — APPROVED** [review]

**Review by jackwotherspoon — APPROVED** [review]
LGTM, one small nit.

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    outputs:
+      releases_created: steps.release-please.outputs.releases_created
+      release_tag: steps.release-please.outputs.tag_name
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: simple
+          packageName: btlr
+          releaseType: simple
+          versionFile: 'cmd/version.txt'
```
Comment (jackwotherspoon) [inline .github/workflows/release.yaml:32]: nit: duplicate `releaseType`'s here. I'd remove `release-type` for naming consistency.

**Pull request merged**